sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```
@@ -4,7 +4,8 @@
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!-- <PackageReference Include="Azure.Core" /> -->
+    <ProjectReference Include="..\..\..\..\sdk\core\Azure.Core\src\Azure.Core.csproj" />
```

```suggestion
    <ProjectReference Include="..\..\Azure.Core\src\Azure.Core.csproj" />
```

I think we can simplify a bit.   Probably want to throw an issue in for the "change back to a package ref" as well.